code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -22,7 +22,28 @@
 		new /obj/item/clothing/mask/gas(src)
 		new /obj/item/clothing/shoes/sandal/white(src)
 
+/obj/structure/closet/secure_closet/roboticist
+	name = "roboticist's locker"
+	req_access = list(access_robotics)
+	icon_state = "secureres1"
+	icon_closed = "secureres"
+	icon_locked = "secureres1"
+	icon_opened = "secureresopen"
+	icon_broken = "secureresbroken"
+	icon_off = "secureresoff"
 
+	New()
+		..()
+		new /obj/item/weapon/storage/backpack(src)

```

Comment:
Shouldn't they have science backpacks?

`new /obj/item/weapon/storage/backpack/science(src)` etc

Reply:
Robotics loadout is standard backpacks apparently.

Reply:
I would assume the fact that Robos don't get science backpacks is an oversight. I'd put science backpacks in their locker, and in their loadouts.

Reply:
Robotics is too special to be counted among the manchildren that is Science. Also those backpacks would look ugly for Roboticist. Special Robotics backpack when?